### PR TITLE
Feature: Create a contract to deploy and update weights for the LBP

### PR DESCRIPTION
**Pull request by codebuster22**
### In this PR:

- Create LBPDeployer contract which can only be accessed by the safe. It have a function to invoke LBPFactory.create() and LBP.updateWeightsGradually() in one call.

### Links
- Issue: #52 
- [Link to design doc](https://docs.google.com/document/d/1fJ8X6EmCAcp9H0YpCRnVDO3FC72MNtwwy5Z35YItTYg/edit#heading=h.ps9oe2oi4ikb)

### CODE REVIEW

File: contracts/utils/LBPDeployer.sol [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    modifier initialized {
+        require(
+            isInitialized,
+            "Deploter: contract not initialized"
+            );
+    }
+
+    constructor (
+            address _LBPFactory,
+            uint256 _swapFeePercentage
```
Comment (codebuster22) [inline contracts/utils/LBPDeployer.sol:47]: add safe as a parameter.

Comment (codebuster22) [inline contracts/utils/LBPDeployer.sol:47]: add safe as a parameter.

Comment (Zitzak) [reply contracts/utils/LBPDeployer.sol:65]: Done

**Comment by Zitzak** [comment]
This PR became obselete

**Pull request closed without merging**